random.js

```diff
@@ -0,0 +1,10 @@
+// Random Number generator
+
```

Comment:
It might help if we explicitly specify in a comment that this function must be used and not modified.

Reply:
Done!